pkg/ytconfig/generator.go

```diff
@@ -155,6 +154,12 @@ func (g *BaseGenerator) fillClusterConnection(c *ClusterConnection, s *ytv1.RPCT
 	c.ClusterName = g.key.Name
 	c.DiscoveryConnection.Addresses = g.getDiscoveryAddresses()
 	g.fillClusterConnectionEncryption(c, s)
+	if len(g.getMasterCachesAddresses()) == 0 {
+		c.MasterCache.Addresses = g.getMasterAddresses()
+	} else {
+		c.MasterCache.Addresses = g.getMasterCachesAddresses()
+	}
```

Comment:
Looks like default behaviour to go to master if master caches is empty. @gritukan is it better somehow to fill config with master addresses for caches for some reason?